components/carbon-transports/websocket/org.wso2.carbon.websocket.transport/src/main/java/org/wso2/carbon/websocket/transport/SubprotocolBuilderUtil.java

```diff
@@ -45,8 +46,12 @@ public static String syanapeSubprotocolToContentType(String subprotocol) {
     }
 
     public static String contentTypeToSyanapeSubprotocol(String contentType) {
-        return SYNAPSE_SUBPROTOCOL_PREFIX + SYNAPSE_CONTENT_TYPE + "'" + contentType +
-                "'" + SYNAPSE_SUBPROTOCOL_SUFFIX;
+        if (contentType.equals(SYNAPSE_CONTENT_TYPE_GRAPHQL_WS)) {
```

Comment:
Introduce a new property called websocket.subprotocol and if that present use it instead of calling this method to derive subprotocol from the content type.